tests/forte/notebook_tutorial/notebook_classes.py

```diff
@@ -0,0 +1,66 @@
+"""This script is a copy of docs/notebook_tutorial/notebook_classes"""
+import os
+import shutil
+from typing import Dict
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+from forte import Pipeline
+from forte.data import DataPack
+from forte.common import Resources, Config
+from forte.processors.base import PackProcessor
+from forte.data.readers import PlainTextReader
+
+
+class MachineTranslationProcessor(PackProcessor):
```

Comment:
This file looks like the same as [docs/notebook_tutorial/notebook_classes.py](https://github.com/asyml/forte/pull/682/files#diff-559847266f2c695d630f6bf1debe112d77fd672add376c7b2e6e4a65803742c7)